docs/topics/which_historical_backend.md

```diff
@@ -0,0 +1,104 @@
+
+Finally, it must be said you can upload any static dataset of your own to HDFS. Then, this data can be analyzed independently of the context datasets created by the platform, or can be used together with them, for instance, for correlation purposes.
+
+[Top](#top)
+
+##MicroStrategy
+This the Business Intelligence system for the IoT Platform. You should use it if your aim is to get business-oriented visualizations and analytics, since MicroStrategy comes with a wide variety of features:
+
+* General data visualization.
+* New data discovery.
+* Geospatial visualizations.
+* Advanced native analytical functions.
+* Enterprise reports.
+* Dashboards.
+
+All the data must be stored in the Analytical Database (MySQL), which handles a private database per FIWARE service (i.e. a tenant of the platform).
+
+[Top](#top)
+
+##CKAN
+If you want to make public your data, there is no doubt: CKAN is the persistence backend where to create your historical datasets since it is one of the most used Open Data platforms in the world. According to the [Open Knowledge Foundation](https://okfn.org/opendata/), making *open* your data means anyone is free to use, reuse and redestribute it. However, using CKAN does not mean all the publised data has to be public, being configurable in a per dataset basis.
+
+Regarding storage capabilities, the data can be uploaded to the FileStore, a simple file manager, or to the DataStore, which is based on [PostgreSQL](http://www.postgresql.org).
+
+Apart from the open data capabilities of CKAN, it must be highlighted the visualization tools it provides in the form of viewers (other [viewers](http://docs.ckan.org/en/latest/maintaining/data-viewer.html) could be added upon request):
+
+* **Text View** and **Image View**. It simply displays the content of text and image files, respectively, within the FileStore. Special formats such as XML or Json have syntax highlighted.
+* **Recline View**. This is the visualization extension used for context datasets created by the platform in the DataStore. The data can be displayed as a grid, as a 2-axis graph or as a geolocation map (if the data contains some geolocation data).
+
+CKAN does not provide any analytical tool
+
+And equals than HDFS, CKAN allows for uploading you own datasets (open or not).
+
+[Top](#top)
+
+##STH Comet
+As its name denotes, this MongoDB-based backend is used for short-term historical persistence. By short-term must be understood months, days or even days.
+
+It exposes a REST API for data querying. FIWARE service and FIWARE service path must be given in the form of Http headers, together with a authentication token. This means this is a tol for private consumption of data.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-iot-stack/pull/48/commits/93452e87445a80dee72b0712650c19246234aeaa